games/k.yaml

```diff
@@ -226,6 +226,26 @@
   - https://i.imgur.com/3QvTDrN.png
   - https://i.imgur.com/G1rezQr.png
 
+- name: Kizare
+  originals:
+  - 'Legend of Zelda - A Link to the Past'
+  type: similar
+  repo: 'https://code.google.com/archive/p/kizare/'
+  url: >-
+    https://web.archive.org/web/20110427084745/http://www.sdltutorials.com/sdl-games/#:~:text=kizare
+  development: halted
+  status: semi-playable
+  langs:
+  - C++
+  frameworks:
+  - SDL
+  - OpenGL
+  - SpiderMonkey
```

Comment:
```suggestion
```
I don't think spidermonkey is relevant here, it is basically a JS engine meaning it runs plain javascript, and is relevant only for packaging purposes and not for development